README.md

```diff
@@ -251,16 +252,10 @@ The `/etc/hosts` equivalent filepath on Windows is:
 
 For more info, see [Hosts at wikipedia](http://en.wikipedia.org/wiki/Hosts_(file)).
 
-## Contributing
-
-* `git clone` this repository
-* `npm install`
-* `bower install`
-
-## Running Test App
+## Running
 
-* `ember server`
-* Visit your app at http://localhost:4200.
+* `ember server --dummy`
```

Comment:
is `--dummy really needed`?

Reply:
no. neither is the **r** in _server_ as far as i can tell.